src/Sql/Sql.Test/ScenarioTests/LedgerDigestUploadTests.ps1

```diff
@@ -0,0 +1,194 @@
+﻿# ----------------------------------------------------------------------------------
+#
+# Copyright Microsoft Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+
+<#
+.SYNOPSIS
+Tests getting default ledger digest upload configuration
+#>
+function Test-GetDefaultLedgerDigestUpload
```

Comment:
For the Get case, we don't have various parameter options:name, resourceID, etc.? Only for Set?

Reply:
Oh it seems like we do but you test everything in one test?